en/asgardeo/docs/guides/users/manage-users.md

```diff
@@ -494,7 +494,58 @@ To filter users by account status:
     "HTTP/1.1 201 Created"
     ```
 
-4: Confirm email or validate OTP (One-Time Password)
+4: Resend email verification code (Optional)
+
+!!! abstract ""
+
+    === "Request format"
+
+        ```curl
+        curl -X 'POST' \
+        'https://api.asgardeo.io/t/<org_name>/api/identity/user/v1.0/resend-code' \
+        -H 'accept: application/json' \
+        -H 'Content-Type: application/json' \
+        -d '{
+            "user": {
+                "username": "<username>",
+                "realm": "DEFAULT"
+            },
+            "properties": [
+                {
+                    "key": "RecoveryScenario",
+                    "value": "EMAIL_VERIFICATION_OTP"
+                }
+            ]
+        }'
+        ```
+    === "Sample request"
+
+        ```
+        curl -X 'POST' \
+        'https://api.asgardeo.io/t/<org_name>/api/identity/user/v1.0/resend-code' \
+        -H 'accept: application/json' \
+        -H 'Content-Type: application/json' \
```

Comment:
Doesn't this API require authorization?

`-H 'Authorization: Bearer <access_token>' ` is missing

Reply:
Fixed